tests/render/test_render_BulletListContentBlock.py

```diff
@@ -0,0 +1,61 @@
+from great_expectations.render.renderer.content_block import BulletListContentBlock
+
+import glob
+import json
+
+
+def test_all_expectations_using_test_definitions():
+    # Fetch test_definitions for all expectations.
+    # Note: as of 6/20/2019, coverage is good, but not 100%
+    test_files = glob.glob(
+        "tests/test_definitions/*/expect*.json"
+    )
+
+    all_true = True
+    failure_count, total_count = 0, 0
+    types = []
+    # Loop over all test_files, datasets, and tests:
+    for filename in test_files:
+        test_definitions = json.load(open(filename))
+        types.append(test_definitions["expectation_type"])
+
+        for dataset in test_definitions["datasets"]:
+
+            for test in dataset["tests"]:
+                # Construct an expectation from the test.
+                fake_expectation = {
+                    "expectation_type": test_definitions["expectation_type"],
+                    "kwargs": test["in"],
+                }
+
+                try:
+                    # Attempt to render it
+                    render_result = BulletListContentBlock.render(
+                        fake_expectation)
+                    # print(fake_expectation)
+
+                    # Assert that the rendered result matches the intended format.
+                    # Note: THIS DOES NOT TEST CONTENT AT ALL.
+                    # Abe 6/22/2019: For the moment, I think it's fine to not test content.
+                    # I'm on the fence about the right end state for testing renderers at this level.
+                    # Spot checks, perhaps?
+                    assert render_result != None
+                    assert type(render_result) == list
+                    for el in render_result:
```

Comment:
Given the broken template above, would be good to try rendering the template as part  of the test to make sure the template references the right params.

Reply:
💯 💯 💯 💯 💯 
💯 💯 💯 💯 💯